lib/puppet/rest/routes.rb

```diff
@@ -8,10 +9,18 @@ def self.ca
       Route.new(api: '/puppet-ca/v1/', default_server: Puppet[:ca_server], default_port: Puppet[:ca_port])
     end
 
+    # Make an HTTP request to fetch the named certificate, using the given
+    # HTTP client.
+    # @param [Puppet::Rest::Client] client the HTTP client to use to make the request
+    # @param [String] name the name of the certificate to fetch
+    # return [String] the PEM-encoded certificate or certificate bundle
```

Comment:
We might want to document what this can raise (and document those exceptions in the methods that it calls)